source/PluginDev.UnitTests/GooglePlayGames/ISocialPlatform/BaseMockPlayGamesClient.cs

```diff
@@ -13,110 +13,133 @@
-
-    internal BaseMockPlayGamesClient() {
-        Authenticated = true;
-    }
-
-    public virtual void Authenticate(System.Action<bool> callback, bool silent) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public bool IsAuthenticated() {
-        return Authenticated;
-    }
-
-    public virtual void SignOut() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual string GetUserId() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual string GetUserDisplayName() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public string GetUserImageUrl() {
-        throw new NotImplementedException("unsupported");
-    }
-
-    public List<Achievement> GetAchievements() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void UnlockAchievement(string achId, Action<bool> callback) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void RevealAchievement(string achId, Action<bool> callback) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void IncrementAchievement(string achId, int steps, Action<bool> callback) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void ShowAchievementsUI() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void ShowLeaderboardUI(string lbId) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void SubmitScore(string lbId, long score, Action<bool> callback) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void LoadState(int slot, OnStateLoadedListener listener) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public virtual void UpdateState(int slot, byte[] data, OnStateLoadedListener listener) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public BasicApi.Multiplayer.IRealTimeMultiplayerClient GetRtmpClient() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public BasicApi.Multiplayer.ITurnBasedMultiplayerClient GetTbmpClient() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public BasicApi.SavedGame.ISavedGameClient GetSavedGameClient() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public void RegisterInvitationDelegate(InvitationReceivedDelegate deleg) {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public Invitation GetInvitationFromNotification() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    public bool HasInvitationFromNotification() {
-        throw new NotSupportedException("unsupported");
-    }
-
-    static void NotSupported() {
-        throw new NotSupportedException("unsupported");
-    }
-}
-}
-
+namespace GooglePlayGames.UnitTests
```

Comment:
Seems there is no new changes in  this file?

Reply:
removed